### ISSUE TITLE
naomi gd boot screen

### ISSUE THREAD

**Issue by motaromk9** [flyinghead/flycast#1735]
Is there any configuration I can do to show at boot that the game is a gd-rom system? Just like on the original hardware. Or if there is no such option, would it be possible to implement it?
![1](https://github.com/user-attachments/assets/291bb7c5-d449-4d63-b6dc-4ce6f29b6e0c)

**Comment by flyinghead** [comment]
No, there's nothing you can do. This is due to some hardware registers not being emulated, which makes the naomi BIOS thinks it's not a GD-ROM system.
This could be fixed in the future.

**Comment by flyinghead** [comment]
Fixed in the latest `dev` build

**Comment by motaromk9** [comment]
Thank you so much for implementing this!
